woocommerce/woocommerce
# CYS - AI Flow: hide X button on the entrepreneur flow

![KQyos5](https://github.com/woocommerce/woocommerce/assets/4463174/3a14e51b-f672-43aa-adc3-fa653663bdf9)
> [!WARNING]  
> This PR is blocked by #46528 


### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





For the `entrepreneur flow`, we want to hide the X button during the AI flow. 

Fixes #46802



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you are on a WooCommerce site where the CYS AI flow works.
2. Visit `/wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store%2Fdesign-with-ai&ref=entrepreneur-signup` 
3. Ensure that the `Tell us a bit more about your business` page is loaded.
4. Ensure that the X isn't visible.
5. Continue to flow.
6. Ensure that the `&ref=entrepreneur-signup` parameter is still present in the URL for each page.
7. Ensure that the X isn't visible.


| Classic Flow | Entrepreneur flow |
|--------|--------|
|![KQyos5.png](https://github.com/woocommerce/woocommerce/assets/4463174/3a14e51b-f672-43aa-adc3-fa653663bdf9)  | ![image](https://github.com/woocommerce/woocommerce/assets/4463174/7cc56972-2a5d-486a-a43d-77a91a64c65d) | 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
